src/ledger.js

```diff
@@ -978,17 +995,28 @@ export class LedgerSignMultisigTransaction extends LedgerBitcoinInteraction {
             transactionVersion: 1, // tx version
           },
         );
-        return this.parse(transactionSignature);
+
+        // If we were passed a PSBT initially, we want to return a PSBT with partial signatures
+        // rather than the normal array of signatures.
+        if (this.psbt) {
+          return addSignaturesToPSBT(this.network, this.psbt, this.pubkeys, this.parseSignature(transactionSignature, "buffer"))
+        } else {
+          return this.parseSignature(transactionSignature, "hex");
+        }
       } finally {
         transport.close();
       }
     });
   }
 
-  parse(transactionSignature) {
```

Comment:
really shouldn't have even been using the `parse` method -- that's not supposed to be used on `DirectKeystoreInteraction` classes.